Automattic/wp-calypso
# Add P2 GitHub connection

#### Changes proposed in this Pull Request

![Markup on 2021-11-10 at 3:02:49 PM](https://user-images.githubusercontent.com/2129455/141118439-ca58c088-1715-45b6-a7f9-d0aa387818a3.png)
Add support for Github connection for P2 hubs.

#### Testing instructions
- Build and push 4015-gh-Automattic/p2
- Navigate to `/marketing/connections/<site-url>` for a P2 hub - there should be a Github connection **(should be available only for automatticians)**
- Navigate to `/marketing/connections/<site-url>` for a P2 site - there should be the regular 3rd party connections (facebook/twitter)
- Navigate to `/marketing/connections/<site-url>` for a regular site - there should be the regular 3rd party connections (facebook/twitter) and make sure they work

For P2 hubs, try to connect/disconnect a Github account.
